src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -38,9 +56,24 @@ export const VolunteerFormCalendar = ({
     mode: 'onBlur',
     defaultValues: {
       isPublic: 1,
+      calendarId: calendarData?.id || '',
+      title: calendarData?.title || '',
+      startDate: calendarData?.start_datetime ? appointments?.dateFrom : undefined,
+      startTime: calendarData?.start_datetime ? appointments?.timeFrom : undefined,
+      endDate: calendarData?.end_datetime ? appointments?.dateTo : undefined,
+      endTime: calendarData?.end_datetime ? appointments?.timeTo : undefined,
```

Comment:
`calendarData?.start_datetime` is used for creating an entry in `appointments` and to check here if appointments can be used? this confuses me. and in addition setting undefined here. what if `appointments?.dateFrom` returns a date or undefined. the four lines in default values can become very short. i will refactor `appointments` in this way.

Reply:
```
startDate: appointments.dateFrom,
startTime: appointments.timeFrom,
endDate: appointments.dateTo,
endTime: appointments.timeTo,
```

we even do not need to chain with `?` because we define `appointments` anyways

Reply:
01b627a